rails-install-ubuntu.sh

```diff
@@ -2,50 +2,69 @@
 
 set -e
 
+echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
+sudo add-apt-repository ppa:chris-lea/node.js
+
+echo "Adding deb. repository for PostgreSQL"
+echo "deb http://apt/postgresql.org/pub/repos/apt/ trusty-pgdg main" > /etc/apt/sources.list.d/pgdg.list"
+wget --quiet -O - http://apt.postgresql.org/pub/repos/apt/ACCC4CF8.asc | sudo apt-key add -
+
 echo "Updates packages. Asks for your password."
 sudo apt-get update -y
 
 echo "Installs packages. Give your password when asked."
-sudo apt-get install build-essential bison openssl libreadline6 libreadline6-dev curl git-core zlib1g zlib1g-dev libssl-dev libyaml-dev libsqlite3-0 libsqlite3-dev sqlite3 postgresql postgresql-contrib libpq-dev pgadmin3 libxml2-dev libxslt-dev autoconf libc6-dev nodejs -y
+sudo apt-get install build-essential git-core curl openssl libssl-dev libcurl4-openssl-dev zlib1g zlib1g-dev libreadline libreadline-dev libreadline6 libreadline6-dev libyaml-dev libsqlite3-dev libsqlite3-0 sqlite3 libxml2-dev libxslt1-dev python-software-properties libffi-dev libgdm-dev libncurses5-dev automake autoconf libtool bison libc6-dev postgresql-common postgresql-9.4 libpq-dev pgadmin3  nodejs -y
 
 echo "Installs ImageMagick for image processing"
 sudo apt-get install imagemagick --fix-missing -y
 
 echo "Installs RVM (Ruby Version Manager) for handling Ruby installation"
 gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
-curl -sSL https://get.rvm.io | bash -s -- --version 1.26.11
+curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
Good catch, I agree that this should be changed back. I don't really know how this could be automated save a website scrape from the rvm website but that is way out of scope for this script.